Bottom: 4ebe2cd5f576b6f51a57b6dc6867be42bc56491d
Top:    cdfc2cd4a24ede9ab361caeb3b55dc7676f1adc4
Author: scott snyder <sss@karma>
Date:   2020-03-16 22:55:24 -0400

SMBC fixes.

Change all SMBC urls to https.



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
index bc003b7..59fc456 100644
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
@@ -41,7 +41,7 @@ public class SMBC extends IndexedComic {
 
 	@Override
 	public String getComicWebPageUrl() {
-		return "http://www.smbc-comics.com/";
+		return "https://www.smbc-comics.com/";
 	}
 
         // Latest ID is no longer in the html.
@@ -117,7 +117,7 @@ public class SMBC extends IndexedComic {
 
 	@Override
 	public String getStripUrlFromId(int num) {
-		return "http://www.smbc-comics.com/?id=" + num;
+		return "https://www.smbc-comics.com/?id=" + num;
 	}
 
 	@Override
@@ -158,7 +158,7 @@ public class SMBC extends IndexedComic {
 		image_url = image_url.replaceAll("\\(","%28");
 		image_url = image_url.replaceAll("\\)","%29");
 		if ( image_url.indexOf("http") == -1 ) {
-			image_url=image_url.replaceAll("^","http://www.smbc-comics.com/");
+			image_url=image_url.replaceAll("^","https://www.smbc-comics.com/");
 		}
 //Log.d("SMBC", "image_url " + image_url);
 		final_title = final_str.replaceAll(".*title=\"","");
